@ecomplus/storefront-template/template/js/lib/utils.js

```diff
@@ -81,17 +80,12 @@ if ($timers.length) {
         } else {
           return clearInterval(stopwatch)
         }
-        $timerCount.text(`${formatTime(hours)}:${formatTime(minutes)}:${formatTime(seconds)}`)
+        $timerCount.innerHTML = `${formatTime(hours)}:${formatTime(minutes)}:${formatTime(seconds)}`
       }
       const stopwatch = setInterval(updateTimerCount, 1000)
       updateTimerCount()
     }
   })
 }
 
-$('#go-to-top').on('click', () => {
-  window.scroll({
-    top: 0,
-    behavior: 'smooth'
-  })
-})
+document.querySelector('#go-to-top').addEventListener('click', () => window.scrollTo(0, 0))
```

Comment:
```suggestion
document.getElementById('go-to-top').addEventListener('click', () => {
  window.scroll({
    top: 0,
    behavior: 'smooth'
  })
})
```

`getElementById` é mais rápido que `querySelector`, e você estava funcionamento (a callback) aqui  também...